lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -431,10 +431,10 @@ def scope_relation(record, relation)
         }
 
         valid_to = record.valid_to.yield_self { |valid_to|
-          # valid_datetime が valid_from ~ valid_to の範囲外だった場合、
+          # レコードを更新する時に valid_datetime が valid_from ~ valid_to の範囲外だった場合、
           #   一番近い未来の履歴レコードを参照して更新する
           # という仕様があるため、それを考慮して valid_to を設定する
-          if (record.valid_datetime && (record.valid_from..record.valid_to).include?(record.valid_datetime)) == false
+          if (record.valid_datetime && (record.valid_from..record.valid_to).include?(record.valid_datetime)) == false && (not record.new_record?)
```

Comment:
I think `record.persisted?` is a better way to do it.

Reply:
Thanks!. I fixed using to `record.persisted? `